alphastats/DataSet_Plot.py

```diff
@@ -95,15 +94,15 @@ def plot_sampledistribution(
         # create long df
         matrix = self.mat if not use_raw else self.rawmat
         df = matrix.unstack().reset_index()
-        df.rename(columns={"level_1": self.sample, 0: "Intensity"}, inplace=True)
+        df.rename(columns={"level_1": Cols.SAMPLE, 0: "Intensity"}, inplace=True)
```

Comment:
Seeing this here, but I think we should replace intensity either with the more generic term abundance, or use what was actually the original name. Intensity or LFQ intensity, or even SILAC ratio makes a bit difference.